src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig

```diff
@@ -3,107 +3,116 @@
-            {% endset %}
-            {% set body_row_cols = body_row_cols|merge([{
-                has_checkbox: true,
-                content: col_raw,
-                raw: true,
-            }]) %}
-
-            {% set col_raw %}
-                {% include '@ibexadesign/ui/location_path.html.twig' with {'locations': location.pathLocations, 'link_last_element': true} %}
-            {% endset %}
-            {% set body_row_cols = body_row_cols|merge([{
-                content: col_raw,
-                raw: true,
-            }]) %}
-
-            {% set body_row_cols = body_row_cols|merge([
-                { content: location.childCount },
-            ]) %}
-
-            {% set col_raw %}
-                <label
-                    class="ibexa-checkbox-icon {{ not location.explicitlyHidden ? 'is-checked' }}{{ not can_hide[location.id] ? 'disabled' }}"
-                    title="{{ 'tab.locations.visibility'|trans|desc('Visibility') }}"
-                >
-                    <svg class="ibexa-icon ibexa-icon--view">
-                        <use xlink:href="{{ ibexa_icon_path('view') }}"></use>
-                    </svg>
-                    <svg class="ibexa-icon ibexa-icon--view-hide">
-                        <use xlink:href="{{ ibexa_icon_path('view-hide') }}"></use>
-                    </svg>
+        {% set body_rows = [] %}
+        {% if locations is not empty %}
+            {% for location in locations %}
+                {% set body_row_cols = [] %}
+
+                {% set col_raw %}
+                    {{ form_widget(form_content_location_remove.locations[location.id], {'attr': {'disabled': not location.canDelete}}) }}
+                {% endset %}
+                {% set body_row_cols = body_row_cols|merge([{
+                    has_checkbox: true,
+                    content: col_raw,
+                    raw: true,
+                }]) %}
+
+                {% set col_raw %}
+                    {% include '@ibexadesign/ui/location_path.html.twig' with {'locations': location.pathLocations, 'link_last_element': true} %}
+                {% endset %}
+                {% set body_row_cols = body_row_cols|merge([{
+                    content: col_raw,
+                    raw: true,
+                }]) %}
+
+                {% set body_row_cols = body_row_cols|merge([
+                    { content: '' },
+                ]) %}
+
+                {% set col_raw %}
                     <input
-                        class="ibexa-checkbox-icon__checkbox"
-                        {{ not location.explicitlyHidden ? 'checked' }}
-                        type="checkbox"
-                        name="updateVisibility"
-                        {{ not can_hide[location.id] ? 'disabled' }}
+                        class="ibexa-input ibexa-input--radio"
+                        type="radio"
```

Comment:
Are you sure? I think that the old type (`checkbox`) is the correct one.
```suggestion
                        type="checkbox"
```

Reply:
Here should be `radio`
![Screenshot 2021-12-30 at 09 23 08](https://user-images.githubusercontent.com/24355391/147734319-38a3ebc5-8f8c-4703-9e3b-f6a051323c78.png)

Reply:
Oh, you are right, I was confused because the order of the columns has been changed. :)